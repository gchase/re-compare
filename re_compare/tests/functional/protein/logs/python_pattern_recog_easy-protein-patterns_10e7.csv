regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,492.48313903808594,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,492.48313903808594,"[-1, -1]"
DHKNLD.,first_match,1,0.9508132934570312,"[613257, 613263]"
DHKNLD.,consecutive,1,0.9508132934570312,"[613257, 613263]"
DHKNLD.,consecutive,2,0.0021457672119140625,"[613397, 613403]"
DHKNLD.,all_matches,2,0.9529590606689453,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,4.894018173217773,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,4.894018173217773,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.014066696166992188,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,4.908084869384766,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,1,0.38886070251464844,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,1,0.38886070251464844,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,2,0.003814697265625,"[215383, 215393]"
(GI)(G|R)P.Y..K..,all_matches,2,0.39267539978027344,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.004291534423828125,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.004291534423828125,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0021457672119140625,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.0064373016357421875,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,4.0340423583984375,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,4.0340423583984375,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.04100799560546875,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,4.075050354003906,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.008821487426757812,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.008821487426757812,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.0069141387939453125,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.015735626220703125,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.9860992431640625,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.9860992431640625,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,2.7031898498535156,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,3.689289093017578,"[-1, -1]"
P..L...KWE.,first_match,1,8.218050003051758,"[4211985, 4211995]"
P..L...KWE.,consecutive,1,8.218050003051758,"[4211985, 4211995]"
P..L...KWE.,consecutive,2,0.009059906005859375,"[4212434, 4212444]"
P..L...KWE.,all_matches,2,8.227109909057617,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,268.85008811950684,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,268.85008811950684,"[-1, -1]"
