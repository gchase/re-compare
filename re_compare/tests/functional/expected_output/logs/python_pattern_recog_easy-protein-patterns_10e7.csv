regex,matchtype,matchnumber,time(ms),span
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,first_match,0,417.0827865600586,"[-1, -1]"
((((Q|R)(I|V)....)(T|C)D..G)(I|V)V.)(H|F)..,all_matches,0,417.0827865600586,"[-1, -1]"
DHKNLD.,first_match,1,1.5521049499511719,"[613257, 613263]"
DHKNLD.,consecutive,1,1.5521049499511719,"[613257, 613263]"
DHKNLD.,consecutive,2,0.0040531158447265625,"[613397, 613403]"
DHKNLD.,all_matches,2,1.5561580657958984,"[-1, -1]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,first_match,1,5.21397590637207,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,1,5.21397590637207,"[98919, 98924]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,consecutive,2,0.014781951904296875,"[99268, 99273]"
(((S|A)(L|I|V|M))(N|G|S))(S|T|A)DD,all_matches,2,5.228757858276367,"[-1, -1]"
(GI)(G|R)P.Y..K..,first_match,1,0.44608116149902344,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,1,0.44608116149902344,"[214952, 214962]"
(GI)(G|R)P.Y..K..,consecutive,2,0.003814697265625,"[215383, 215393]"
(GI)(G|R)P.Y..K..,all_matches,2,0.44989585876464844,"[-1, -1]"
G|S|T|A|D|N|E|K|R,first_match,1,0.005245208740234375,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,1,0.005245208740234375,"[3, 3]"
G|S|T|A|D|N|E|K|R,consecutive,2,0.0040531158447265625,"[4, 4]"
G|S|T|A|D|N|E|K|R,all_matches,2,0.009298324584960938,"[-1, -1]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),first_match,1,4.75001335144043,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,1,4.75001335144043,"[49973, 49978]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),consecutive,2,0.041961669921875,"[50386, 50391]"
((((L|I|V|M|F|G|AE)(L|I|M|S|T|A|C))(G|S)G)(K|N|L|M))(S|A|D|N),all_matches,2,4.791975021362305,"[-1, -1]"
(N|S.TNH.Y)(F|W),first_match,1,0.009059906005859375,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,1,0.009059906005859375,"[122, 123]"
(N|S.TNH.Y)(F|W),consecutive,2,0.010967254638671875,"[441, 442]"
(N|S.TNH.Y)(F|W),all_matches,2,0.02002716064453125,"[-1, -1]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),first_match,1,0.61798095703125,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,1,0.61798095703125,"[11709, 11713]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),consecutive,2,2.2208690643310547,"[58929, 58933]"
((((I|V.DS)(G|A|S))(G|A|S|C))(G|A|S|T))(G|A),all_matches,2,2.8388500213623047,"[-1, -1]"
P..L...KWE.,first_match,1,8.751869201660156,"[4211985, 4211995]"
P..L...KWE.,consecutive,1,8.751869201660156,"[4211985, 4211995]"
P..L...KWE.,consecutive,2,0.0059604644775390625,"[4212434, 4212444]"
P..L...KWE.,all_matches,2,8.757829666137695,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,first_match,0,204.5879364013672,"[-1, -1]"
(G|A.)(L|I|V|M)..HGTD,all_matches,0,204.5879364013672,"[-1, -1]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),first_match,1,0.007867813110351562,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,1,0.007867813110351562,"[830, 838]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),consecutive,2,0.003814697265625,"[996, 1004]"
((((((P(S|A|P))(L|I|V))(D|N|H))(A|C|D|E|F|H|I|M|P|Q|R|S|T|V|W|Y))(A|C|D|E|G|H|I|K|L|M|N|P|Q|R|S|T|V|W|Y))(A|C|D|E|F|G|H|I|K|L|M|N|P|Q|R|T|V|W|Y)S)(A|E|F|G|I|K|L|M|N|Q|R|S|T|V|W|Y),all_matches,2,0.011682510375976562,"[-1, -1]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),first_match,1,2.4051666259765625,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,1,2.4051666259765625,"[41351, 41357]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),consecutive,2,3.004312515258789,"[130717, 130723]"
((((E|D|Q|H)(A|C|D|E|F|G|H|I|L|M|N|P|Q|R|S|T|V|W|Y)K)(A|C|F|G|H|K|L|M|N|P|Q|R|S|T|W|Y))(D|N)G)(A|C|D|E|F|H|I|K|M|P|Q|R|S|T|V|W),all_matches,2,5.409479141235352,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),first_match,0,15.228271484375,"[-1, -1]"
((((DG)(P|D)SA)(G|S))(L|I|V|M|C|A))(T|A),all_matches,0,15.228271484375,"[-1, -1]"
A|G....G,first_match,1,0.006198883056640625,"[15, 15]"
A|G....G,consecutive,1,0.006198883056640625,"[15, 15]"
A|G....G,consecutive,2,0.0040531158447265625,"[19, 19]"
A|G....G,all_matches,2,0.010251998901367188,"[-1, -1]"
((CP|.)(S|T)N)(I|L|V)G,first_match,1,3.201007843017578,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,1,3.201007843017578,"[57782, 57786]"
((CP|.)(S|T)N)(I|L|V)G,consecutive,2,0.014066696166992188,"[58001, 58005]"
((CP|.)(S|T)N)(I|L|V)G,all_matches,2,3.2150745391845703,"[-1, -1]"
